cdc/sink/codec/interface.go

```diff
@@ -39,8 +39,8 @@ type EventBatchEncoder interface {
 	Build() []*MQMessage
 	// Size returns the size of the batch(bytes)
 	Size() int
-	// SetParams provides the encoder with more info on the sink
-	SetParams(params map[string]string) error
+	// setParams provides the encoder with more info on the sink
```

Comment:
I'm wondering if it really needs to be extracted as an interface? Because it seems like some protocols don't need it?